lib/gnat/jetstream/jetstream.ex

```diff
@@ -10,7 +10,9 @@ Sends `AckAck` acknowledgement to the server.
 Acknowledges a message was completely handled.
 """
 @spec ack(message :: Gnat.message()) :: :ok
-def ack(message)
+def ack(%{reply_to: nil}) do
```

Comment:
I ran into a case where I got a `%{status: "408", reply_to: nil}` message at the end of the stream and my code tried to ack the message which resulted in a TCP error and it killed my gnat connection. So I thought it would be good to validate the `reply_to` here with a more useful error message